source/Point.cpp

```diff
@@ -334,7 +334,7 @@ Point Point::Unit() const
 	double b = LengthSquared();
 	if(!b)
 		return Point(1., 0.);
-	return *this / sqrt(b);
+	return *this * (1. / sqrt(b));
```

Comment:
Out of curiosity, what's the benefit of doing it this way?

Reply:
Multiplication is faster in the hardware than division.

Reply:
Shouldn't the compiler optimize it anyway?

Reply:
I don't know. The point division operator is custom, so I'm not sure it can determine that it does the same thing.

Reply:
If it doesn't make any difference I'd prefer the code to be simple, but I guess it's fine to leave it as multiplication since it's already done this way in the non-SSE version.